test/unit/StubbedStreamrClient.ts

```diff
@@ -1,19 +1,23 @@
-import { StreamrClient } from '../../src/'
+import { StreamrClient } from '../../src/index'
 import { Stream } from '../../src/stream'
 
 export default class StubbedStreamrClient extends StreamrClient {
+    // @ts-expect-error
+    // eslint-disable-next-line class-methods-use-this
     getUserInfo() {
         return Promise.resolve({
             username: 'username',
         })
     }
 
-    async getStream () {
-        return new Stream(null, {
+    // eslint-disable-next-line class-methods-use-this
+    async getStream() {
+        return new Stream(null as any, {
             id: 'streamId',
             partitions: 1,
         })
     }
 }
 // publisherId is the hash of 'username'
+// @ts-expect-error
```

Comment:
If it's not too much effort maybe add additional comment after `// @ts-expect-error` explaining what it's erroring on? Might make these more likely to get fixed if the comment gives a clue as to what's wrong at a glance.

e.g.
```typescript 
 // @ts-expect-error additional commenting
 ```

Reply:
I'll make another PR where most of these will be fixed, and the rest commented.